controllers/providers/aws/aws.go

```diff
@@ -55,7 +54,8 @@ var (
 )
 
 const (
-	IAMPolicyPrefix = "arn:aws:iam::aws:policy"
+	IAMPolicyPrefix             = "arn:aws:iam::aws:policy"
+	FargateProfileStatusMissing = "MISSING"
```

Comment:
Can we not use a string for checking that it's missing? if we can do a nil check instead that would be nicer.

Reply:
I can do that but it will require passing the profile status as a pointer so I can check nil.  You OK with that?

Reply:
will use aws.StringValue(nil) instead of FargateProfileStatusMissing